localization/pose_initializer/src/pose_initializer/ndt_localization_trigger_module.cpp

```diff
@@ -0,0 +1,75 @@
+// Copyright 2022 The Autoware Contributors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.S
+#include "ndt_localization_trigger_module.hpp"
+
+#include <component_interface_specs/localization.hpp>
+#include <component_interface_utils/rclcpp/exceptions.hpp>
+
+#include <memory>
+
+using ServiceException = component_interface_utils::ServiceException;
+using Initialize = localization_interface::Initialize;
+
+NdtLocalizationTriggerModule::NdtLocalizationTriggerModule(rclcpp::Node * node)
+: logger_(node->get_logger())
+{
+  client_ekf_trigger_ = node->create_client<SetBool>("ekf_trigger_node");
+  client_ndt_trigger_ = node->create_client<SetBool>("ndt_trigger_node");
+}
+
+void NdtLocalizationTriggerModule::deactivate() const
```

Comment:
How about merging activate and deactivate functions in localization_trigger_module since they are almost the same?

Reply:
That's better. Merged.